### ISSUE TITLE
Wordmove integration?

### ISSUE THREAD

**Issue by VesterDe** [wckr/wocker#51]
Hey, I'm not knowledgeable enough to know if this is possible, or how difficult it would be to implement, but could [Wordmove](https://github.com/welaika/wordmove) be somehow build into Wocker? I think if that were possible it would make Wocker the #1 dev tool for me (it is anyway, but remote push/pull is the bottleneck).

I would be willing to research and try to make some modification if I got a bit of input where to go with this question.

**Comment by ixkaito** [comment]
@VesterDe 

Thanks for using Wocker!

I know Wordmove and it's possible to include it in Wocker. I have a plan to add Xdebug, PHPUnit, MailCatcher and Wordmove to Wocker, and I'm already adding Xdebug and PHPUnit on the develop branch. MailCatcher and Wordmove are on the bottom of my todo list because they need Ruby which Wocker doesn't have and will make the image size larger. Please wait until then or you can do it yourself, and I'm glad if you send a pull request.

**Comment by VesterDe** [comment]
I've managed to get ruby and Wordmove working locally (although without a wocker-cli implementation, just inside the individual docker shells) I've already tested pulling and pushing plugins and themes and they both seem to work.

The issue that remains is the database. I get an error that is outlined [here](https://github.com/welaika/wordmove/wiki/invalid-byte-sequence-in-UTF-8-while-pushing---pulling-db) and the solution doesn't really apply to the environment we're working with. An alternate solution is to edit Wordmove slightly, but that's also not advisable.

It's a bit out of my depth, but if anyone has any input I'd be happy to keep working at it. Pulling and pushing files is already a great step I think. Perhaps we can integrate it step by step?
